compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -243,6 +257,9 @@ class SkikoComposeUiTest(
         while (!isIdle()) {
             renderNextFrame()
             uncaughtExceptionHandler.throwUncaught()
+            if (!areAllResourcesIdle()) {
+                delay(IDLING_RESOURCES_CHECK_INTERVAL_MS)
```

Comment:
so JS tests should avoid using "waitForIdle" and use "awaitIdle" instead withit TestScope (coroutine scope). Then idling resources should work there too, right?

Reply:
That's what I was thinking, but the problem is that all the automatic synchronization (following `setContent` and when you do `onNode(...)` calls `waitForIdle`. So to use it with JS we'll need a new, async, API for those calls.